inc/umock_c/umock_c_internal.h

```diff
@@ -23,6 +23,12 @@
 #include "umock_c/umockstring.h"
 #include "umock_c/umockautoignoreargs.h"
 
+// For gcc build that causes a false negative
+#ifndef _MSC_VER
```

Comment:
You are not going to like this, but shouldn't this be #ifdef GCC (or something that says "if compiling with gcc ... ".

Right now it says: "if it is not Visual Studio..." (so it applies to gcc, apple, clang, intel etc all treated the same).

Reply:
Like in line 34...

Reply:
I was unaware of the GCC, will change.